PrivateClinicsWebNet.DataAccess/Services/JwtTokenService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using PrivateClinicsWebNet.DataAccess.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PrivateClinicsWebNet.DataAccess.Services
+{
+    public class JwtTokenService : ITokenService
+    {
+        private readonly IConfiguration _configuration;
```

Comment:
Use IOptions pattern because if u inject IConfiguration then ur class can use all configuration not related to this class